src/window.blp

```diff
@@ -96,6 +96,15 @@ template $Window: Adw.ApplicationWindow {
               view: tab_view;
             }
 
+            [top]
+            Revealer revealer_load_bar {
+              transition-duration: 300;
+              transition-type: slide_down;
+              ProgressBar load_bar {
+                styles ["osd"]
+              }
+            }
+
```

Comment:
Okay yeah ig I can make the progress bar transparent.  Also I thought it might be useful to show the progress bar for offline docs for example when opening new tabs and it's trying to load something, you just see a blank screen for a good second and it might feel a bit unresponsive.

I feel like we should keep the progress bar for offline docs as well. It might be helpful in future as well if we support other offline docs that take longer to load.